src/Factory/BookFactory.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Factory;
+
+use App\Entity\Book;
+use App\Repository\BookRepository;
+use Zenstruck\Foundry\RepositoryProxy;
+use Zenstruck\Foundry\ModelFactory;
+use Zenstruck\Foundry\Proxy;
+
+/**
+ * @extends ModelFactory<Book>
+ *
+ * @method static Book|Proxy createOne(array $attributes = [])
+ * @method static Book[]|Proxy[] createMany(int $number, array|callable $attributes = [])
+ * @method static Book|Proxy find(object|array|mixed $criteria)
+ * @method static Book|Proxy findOrCreate(array $attributes)
+ * @method static Book|Proxy first(string $sortedField = 'id')
+ * @method static Book|Proxy last(string $sortedField = 'id')
+ * @method static Book|Proxy random(array $attributes = [])
+ * @method static Book|Proxy randomOrCreate(array $attributes = [])
+ * @method static Book[]|Proxy[] all()
+ * @method static Book[]|Proxy[] findBy(array $attributes)
+ * @method static Book[]|Proxy[] randomSet(int $number, array $attributes = [])
+ * @method static Book[]|Proxy[] randomRange(int $min, int $max, array $attributes = [])
+ * @method static BookRepository|RepositoryProxy repository()
+ * @method Book|Proxy create(array|callable $attributes = [])
+ */
+final class BookFactory extends ModelFactory
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        // TODO inject services if required (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#factories-as-services)
+    }
+
+    protected function getDefaults(): array
+    {
+        $categories = ['History', 'Politics', 'Ecology', 'Feminism', 'Antiracism', 'Military', 'Novel', 'Illustrated'];
+        
+        return [
+            'isbn' => self::faker()->text(10),
+            'title' => self::faker()->realText(40),
+            'writerFirstName' => self::faker()->firstName(),
+            'writerLastname' => self::faker()->lastName(),
+            'category' => $categories[array_rand($categories)],
```

Comment:
Using array_rand() is a bad idea because the random number generator will not be seeded, so a different value will be injected at every run, so it will be difficult to assert in your tests because the data will always change.

Use `CategoryFactory::random()` instead (which is biased): https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#using-your-factory